chatapi_ros/README.md

```diff
@@ -0,0 +1,90 @@
+# chatapi_ros
+The purpose of this repository is to make existing chat APIs usable with ROS.
+
+## Setting up environment
+Please install following packages.
+```
+sudo pip install google-api-python-client
+sudo pip install oauth2client
+sudo pip install pya3rt
+```
+
+Please install ros_speech_recognition.
+```
+sudo apt install ros-${ROS_DISTRO}-ros-speech-recognition
+```
+%You can get the latest version from [here](https://github.com/jsk-ros-pkg/jsk_3rdparty/tree/master/ros_speech_recognition). Please try the latest version if it does not work well.
+
+## Set API Keys for [Chaplus](http://www.chaplus.jp/api) and [A3RT](https://a3rt.recruit-tech.co.jp/product/talkAPI)
+### Get Chaplus APIKey
+Please access [here](http://www.chaplus.jp/api#contact) and enter your information.
+
+### Get A3RT APIKey
+Please access [here](https://a3rt.recruit-tech.co.jp/product/talkAPI/registered/) and enter your information.
+
+### Write your keys in `settings.yaml`
+```
+chaplusAPIKEY: xxxxxxxxxx
+a3rtAPIKEY: yyyyyyyyyy
+```
+
+## Get json file for Google speech recognition API.
+### Get json file
+Please follow the step 1 and 2 in [here](https://cloud.google.com/speech-to-text/docs/quickstart-protocol) and download json file.
```

Comment:
please put this information to https://github.com/jsk-ros-pkg/jsk_3rdparty/blob/master/ros_speech_recognition/README.md